internal/lib/stats/cpu_metrics.go

```diff
@@ -0,0 +1,113 @@
+package statsserver
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cri-o/cri-o/internal/config/cgmgr"
+	"github.com/cri-o/cri-o/internal/lib/sandbox"
+	"github.com/cri-o/cri-o/internal/oci"
+	"github.com/sirupsen/logrus"
+	types "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func GenerateSandboxCPUMetrics(sb *sandbox.Sandbox, c *oci.Container, stats interface{}) []*types.Metric {
```

Comment:
No need to have `stats` being an `interface{}` here, please use a concrete type.